src/js/hca/utils/validations.js

```diff
@@ -88,7 +88,11 @@ function isValidDate(day, month, year) {
 }
 
 function isValidName(value) {
-  return /^[a-zA-Z][a-zA-Z '\-]*$/.test(value);
+  return /^[a-zA-Z '\-]*$/.test(value);
+}
+
+function isValidMinimumName(value) {
```

Comment:
I was thinking about that, but the validation needs to be different if it's first name or last name. Wasn't sure how to handle that, since this function accepts a value and not the whole field, there's not a way to have conditional logic within it to check with field it is?